x-pack/filebeat/processors/decode_cef/cef/cef_test.go

```diff
@@ -523,3 +523,13 @@ func TimestampField(v string) *Field {
 	}
 	return &Field{String: v, Type: TimestampType, Interface: ts}
 }
+
+func FuzzUnpack(f *testing.F) {
+	f.Add(headerOnly)
+	f.Add(escapedMessage)
+
+	f.Fuzz(func(t *testing.T, data string) {
+		var e Event
+		e.Unpack(data)
+	})
+}
```

Comment:
This one is no longer necessary as the existing fuzz test has been modernized in https://github.com/elastic/beats/pull/45583.